tsl/test/sql/chunk_api.sql

```diff
@@ -149,6 +149,7 @@ FROM (
 GRANT USAGE
    ON FOREIGN SERVER :DATA_NODE_1, :DATA_NODE_2
    TO :ROLE_1, :ROLE_DEFAULT_PERM_USER;
+GRANT CREATE ON SCHEMA public TO :ROLE_1;
```

Comment:
This is also set in `test/runner.sh` so it shouldn't be necessary to set it here.

Reply:
I don't know why tests fails when we grant required permissions to these roles in runner.sh.
My guess is that we don't grant permission on a foreign server to these roles in runner.sh.
Please correct me if iam wrong.

Reply:
I have created a to #4856 track it.